src/App.tsx

```diff
@@ -1,61 +1,98 @@
 import './App.scss';
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const App: React.FC = () => {
+  const [todos, setTodos] = useState(todosFromServer);
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [newTodoTitle, setNewTodoTitle] = useState('');
+  const [errorVisiableUser, setErrorVisiableUser] = useState(false);
+  const [errorVisiableTitle, setErrorVisiableTitle] = useState(false);
+
+  const handleSuccess = (userId: number) => {
```

Comment:
you can use selectedUserId on method

```suggestion
  const handleSuccess = () => {
```

Reply:
but you can expect here event and call preventDefault 
```suggestion
  const handleSuccess = (event) => {
```